ibm_mq/tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+import subprocess
+import copy
+import docker
+import logging
+
+from datadog_checks.dev import docker_run  # , temp_dir
+from datadog_checks.ibm_mq import IbmMqCheck
+
+from . import common
+
+log = logging.getLogger(__file__)
+
+
+@pytest.fixture
+def check():
+    return IbmMqCheck('ibm_mq', {}, {})
+
+
+@pytest.fixture
+def instance():
+    inst = copy.deepcopy(common.INSTANCE)
+    container = subprocess.check_output([
+        'docker',
+        'ps',
+        '-q'
+    ])
+    container = container.decode("utf-8").strip()
+
+    inst['docker_container'] = container
+
+    log.warning(container)
+    client = docker.from_env()
```

Comment:
Please let's not use docker lib since it's not actually a dep and we should try not to put test logic in main code. Instead simply parse `docker-compose ps`

Reply:
I removed this it was stale from some testing